content/About/Performance/home.pml

```diff
@@ -1,12 +1,200 @@
 <!-- <title>seL4 benchmarks</title> -->
 <!--
-    Copyright 2020 seL4 Project a Series of LF Projects, LLC.
-    SPDX-License-Identifier: CC-BY-SA-4.0
+Copyright 2020 seL4 Project a Series of LF Projects, LLC.
+SPDX-License-Identifier: CC-BY-SA-4.0
 -->
-<h1>
-  Performance
-</h1>
-<p>
-  This page displays the latest benchmark numbers for seL4 from the
-  publicly available sel4bench repository.
-</p><%@ include file="res.html"%>
+
+<h1>Performance</h1>
+<p>This page displays the latest benchmark numbers for seL4 from the publicly
+available <a href="https://github.com/seL4/sel4bench">sel4bench repository</a>.
+The following times are measured and reported as mean with standard deviation,
+both rounded to the nearest integer.</p>
+<ul><li><b>IRQ invoke</b>: Time in cycles to invoke a user-level interrupt handler running in the same
+address space as the interrupted thread.</li>
+<li><b>IPC call</b>: Time in cycles for invoking a server in a different address space on the same core.</li>
+<li><b>IPC reply</b>: Time in cycles for a server replying to a client in a different address space on
+the same core.</li>
+</ul><h2>Default</h2>
+<table style="border-style: solid none solid none; border-color: black; border-width: 0px;; border-collapse: collapse; width: 100%; word-wrap: break-word;">
```

Comment:
Can styles be added to a stylesheet instead of being inline here?

Reply:
I mainly regenerated what was there before, but it would not be too hard to add them to the main style sheet, I think.